### ISSUE TITLE
Investigate QR selection again

### ISSUE THREAD

**Issue by laurjpeterson** [department-of-veterans-affairs/caseflow#10231]
March selected only 135 appeals, rather than 137. Let's investigate why that happened again.

**Comment by laurjpeterson** [comment]
@lowellrex @aroltsch - FYI. no action now, we can discuss during sprint planning.

**Comment by aroltsch** [comment]
Is it for legacy or AMA appeals?

**Comment by aroltsch** [comment]
Ok, I think I got to the bottom of it. We have to add a check here to check if JudgeCaseReview is a legacy case review and only count legacy records for this particular count. https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/judge_case_review.rb#L75
AMA QR selection doesn't happens here. 

In March, we marked two AMA judge case reviews to be selected for a legacy QR.

AMA judge case reviews mistakenly marked as quality_review in March:
```
JudgeCaseReview.where(location: :quality_review).where(created_at: Date.parse('01/03/2019')..Date.parse('31/03/2019')).select { |r| !r.legacy? }.count
=> 2
```  
Legacy judge case reviews in March:

```
JudgeCaseReview.where(location: :quality_review).where(created_at: Date.parse('01/03/2019')..Date.parse('31/03/2019')).select { |r| r.legacy? }.count
=> 135
```
